panos_upgrade_assurance/firewall_proxy.py

```diff
@@ -455,14 +428,18 @@ def get_support_license(self) -> dict:
         - the support level.
         """
         result = {}
-        response = self.op_parser(cmd="request support check", return_xml=True)
-
-        result["support_expiry_date"] = response.findtext(
-            "./SupportInfoResponse/Support/ExpiryDate"
-        )
-        result["support_level"] = response.findtext(
-            "./SupportInfoResponse/Support/SupportLevel"
-        )
+        try:
+            response = self.op_parser(cmd="request support check", return_xml=True)
+        except PanXapiError as exp:
```

Comment:
The connectivity exception occurred with the `PanDeviceXapiError` exception which might narrow down the except a bit more. For the other uncaught exceptions, they will be raised as-is anyway.